src/Glpi/Console/Migration/FormCreatorPluginToCoreCommand.php

```diff
@@ -56,6 +57,24 @@ public function getDescription(): string
     #[Override]
     public function getMigration(): AbstractPluginMigration
     {
-        return new FormMigration($this->db, FormAccessControlManager::getInstance());
+        return new FormMigration(
+            $this->db,
+            FormAccessControlManager::getInstance(),
+            $this->input->getOption('specific-forms')
+        );
+    }
+
+    #[Override]
+    protected function configure()
+    {
+        parent::configure();
+
+        $this->addOption(
+            'specific-forms',
+            null,
+            InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
+            __('Import only specific forms by their IDs'),
```

Comment:
I think `form-id` would be a preferable name:
 - `bin/console migration:formcreator_plugin_to_core --form-id=5`
 - `bin/console migration:formcreator_plugin_to_core --form-id=12 --form-id=23 --form-id=24`

```suggestion
            'form-id',
            null,
            InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
            __('Import only specific forms with the given IDs'),
```